### DISCUSSION TITLE
Please provide the old docs for tailwind.primevue

### DISCUSSION THREAD

**Discussion by ludwig-loth** [primefaces/community#2443] [PrimeVue]
I can't upgrade my project for now because it's in a very hot stage right now. I have to fix bugs and absolutely no time to upgrade to a new version with breaking changes. My project is too big at this point to migrate before my deadline for version 1.0 is due. Maybe after that I cant upgrade, but not know.

My project is heavily relying on the Stepper component, and that is now entirely changed. It would take so much time so migrate this.

Who came up with the idea of upgrading the docs to the new version with breaking changes and not providing the last build of the docs for version 3?
Just add versioning to tailwind.primevue.org like its on primevue.org.

At this point, I really don't know if I'm using PrimeVue in my next project.
Sorry I don't want to sound insulting or something. But this really throws me back in my time management

**Comment by valtteriluomapareto** [comment]
Hey, we are in the same situation at the moment. Apparently the unstyled mode is [not yet supported in v4](https://primevue.org/guides/migration/v4/) (?), so we REALLY need the v3 docs still.

**Comment by AnthonyRuelle** [comment]
Up, same situation here ... V4 not stable ... not ready with tailwind and presets ..

**Comment by AnthonyRuelle** [comment]
Up up 
a lot of components have no style, it doesn't look very stable

**Comment by cagataycivici** [comment]
Hi,

v3 tag is here, you can run it locally, with npm i, npm run dev.

Having said that, new version is switching to @apply rather than pt.

https://github.com/primefaces/primevue-tailwind/tree/main/assets/styles/presets/aura
